### PR TITLE
chore: update dockerfile

### PR DESCRIPTION
**Pull request by wojtekzyla**
# Description

Update UID in docker container from root to 1000

## Type of change

Please delete options that are not relevant.

- [x] Refactor/improvement

## How Has This Been Tested?

Manual and integration tests 

## Checklist

- [x] My commit message is [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] I have run pre-commit on all files before creating the PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by ajasnosz — COMMENTED** [review]
Update changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 ### Changed
 - add docker compose deployment
 - update default microk8s to 1.30
+- change group and user ID in Docker image to 10001:10001
+- create and/or change ownership of /tmp and /.pysnmp to 10001:10001 in Dockerfile
+- while mounting volumes in docker compose explicitly mount the as read only or read write
```
Comment (ajasnosz) [inline CHANGELOG.md:10]: typo with `the` I think there should be `them`

**Review by ajasnosz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (wojtekzyla) [reply CHANGELOG.md]: done

**Pull request merged**